Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -208,6 +208,19 @@ extension SupportedPlatform {
         /// - Since: First available in PackageDescription 5.1
         @available(_PackageDescription, introduced: 5.1)
         public static let v10_15: MacOSVersion = .init(string: "10.15")
+
+        /// The value that represents macOS 10.16, which is treated
+        /// as an alias for macOS 11.0.
+        ///
+        /// - Since: First available in PackageDescription 5.3
+        @available(_PackageDescription, introduced: 5.3)
```

Comment:
Can we mark this deprecated + renamed so that developers get a fixit?

Reply:
I think that's what we want to do but as a separate commit/PR.